docs/life_of_a_godwoken_transaction.md

```diff
@@ -367,25 +356,32 @@ If there is one particular account lock or backend you like, please do not hesis
 ```
 struct RawWithdrawalRequest {
     nonce: Uint32,
+    // chain id
+    chain_id: Uint64,
+    // CKB amount
     capacity: Uint64,
+    // SUDT amount
     amount: Uint128,
     sudt_script_hash: Byte32,
+    // layer2 account_script_hash
     account_script_hash: Byte32,
-    sell_amount: Uint128,
-    sell_capacity: Uint64,
+    // withdrawal registry ID
+    registry_id: Uint32,
+    // layer1 lock to withdraw after challenge period
     owner_lock_hash: Byte32,
-    payment_lock_hash: Byte32,
+    // withdrawal fee, paid to block producer
+    fee: Uint128,
 }
 
 vector WithdrawalRequestVec <WithdrawalRequest>;
 
 struct WithdrawalRequest {
     raw: RawWithdrawalRequest,
-    signature: Signature,
+    signature: Bytes,
 }
 ```
 
-Please refer to [this file](https://github.com/nervosnetwork/godwoken/blob/v0.1.0/crates/types/schemas/godwoken.mol#L71-L94) for a more complete definition and related types.
+Please refer to [this file](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L136-L160) for a more complete definition and related types.
```

Comment:
The full definition and related types can be found in [this file](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L136-L160).